warehouse/templates/manage/manage_base.html

```diff
@@ -79,44 +79,43 @@ <h3 class="sidebar-section__title">Your account</h3>
   slug,
   context,
   confirm_button_label=None,
-  index=None,
   extra_fields=None,
   extra_description=None,
   action=None,
   warning=True,
   custom_warning_text="",
   confirm_string_in_title="True")
 %}
-  <div id="{{ slug }}" class="modal" data-controller="confirm">
+  <div id="{{ slug }}" class="modal" data-controller="confirm modal-close">
     <div class="modal__content" role="dialog">
+      <a href="#modal-close" data-action="click->modal-close#cancel" title="{% trans %}Close{% endtrans %}" class="modal__close">
+        <i class="fa fa-times" aria-hidden="true"></i>
+        <span class="sr-only">{% trans %}close{% endtrans %}</span>
+      </a>
       <form method="POST" class="modal__form" action="{{ action or request.current_route_path() }}">
         <input name="csrf_token" type="hidden" value="{{ request.session.get_csrf_token() }}">
         {{ extra_fields if extra_fields else '' }}
-        <a href="#modal-close" data-action="click->confirm#cancel" title="Close" class="modal__close">
-          <i class="fa fa-times" aria-hidden="true"></i>
-          <span class="sr-only">Close</span>
-        </a>
         <div class="modal__body">
           <h3 class="modal__title">{{ title }}{% if confirm_string_in_title %} {{ confirm_string }}{% endif %}?</h3>
           {% if warning %}
           <div class="callout-block callout-block--danger callout-block--bottom-margin no-top-margin">
             <p>
-              <i class="fa fa-exclamation-triangle" aria-hidden="true"><span class="sr-only">Warning</span></i>
-              This action cannot be undone!
+              <i class="fa fa-exclamation-triangle" aria-hidden="true"><span class="sr-only">{% trans %}Warning{% endtrans %}</span></i>
+              {% trans %}This action cannot be undone!{% endtrans %}
             </p>
             {% if custom_warning_text %}{{ custom_warning_text }}{% endif %}
           </div>
           {% endif %}
-          {% set context = 'your' if confirm_name.lower() == 'username' else 'the' %}
+          {% set context = gettext('your') if confirm_name.lower() == gettext('username') else gettext('the') %}
           {{ extra_description if extra_description else '' }}
-          <p>Confirm {{ context }} {{ confirm_name|lower }} to continue.</p>
+          <p>{{ gettext("Confirm %(context) %(confirm_name) to continue.")|format(context=context, confirm_name=confirm_name.lower()) }}</p>
```

Comment:
@nlhkabu I wasn't sure if this is the right way to get translations on a parametrized message. I'm following https://jinja.palletsprojects.com/en/2.10.x/extensions/#id2 but I think [in the PR you mentioned](https://github.com/pypa/warehouse/pull/6598/files#diff-be3f1c8f146e973b06e40074eb56265cR23) there's a slightly different way? I'm not sure if they're equivalent.

Reply:
Nevermind that doesn't actually work.